Scripts/rename_TestOutput.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
```

Comment:
was this script from a short-term test or will it be useful in future?

Reply:
This script we can run optionally to rename the config field to "MDC2020ad_test2".
It's a convineient feature to run test stages that are tagged as "tests".
I can see us using it for other campaigns as well.